docs/installation/config/openzaak_config.rst

```diff
@@ -301,3 +301,70 @@ endpoint is accessible at ``{{base_url}}/catalogi/api/v1/zaaktypen`` (where
     A GET request to the Catalogi API using Postman
 
 .. _Postman: https://www.getpostman.com/
+
+
+Using external ZGW APIs
+=======================
```

Comment:
I think this should be its own document - probably under the "Guides" TOC (https://github.com/open-zaak/open-zaak/blob/main/docs/installation/index.rst)

Additionally, this documentation page starts by pointing out the CLI-based configuration, as point-and-click configuration is not the best solution. That makes it that this part of the docs is not obvious to get by, or results in a very scattered navigation between documentation pages.